test/EnvironmentManagerTests/EnvironmentManagerCentennialTests.cpp

```diff
@@ -104,7 +104,7 @@ namespace WindowsAppSDKEnvironmentManagerTests
         // Remove the value
         // setting the value to empty is the same as deleting the variable
         VERIFY_NO_THROW(environmentManager.SetEnvironmentVariable(c_EvKeyName, L""));
-        VERIFY_ARE_EQUAL(0, ::GetEnvironmentVariable(c_EvKeyName, nullptr, 0));
+        VERIFY_ARE_EQUAL(0u, ::GetEnvironmentVariable(c_EvKeyName, nullptr, 0));
```

Comment:
Nit: Changes don't seem relevant to the GetPackageInfo fix here. Ditto for similar below.

Reply:
The build was getting messy producing lots of warnings causing me some angst to overlook a problem at first. So I took a pass cleaning most of the warnings (simple things that cut a lot of noise), so I could clearly verify my fix.